requirements/frozen_docs.py3.txt

```diff
@@ -6,6 +6,7 @@
 adal==1.2.1
 aenum==2.1.2
 alabaster==0.7.12
+allure-pytest
```

Comment:
This is only added to support wt.pf git install for PRT, will be removed for review/merge.